index.html

```diff
@@ -220,54 +276,369 @@
+
+            @media screen and (max-width: 768px) {
+                grid-template-columns: 325px;
+                grid-template-rows: 245px 245px 245px;
+                margin-bottom: 95px;
+            }
+        }
+
+        .feedback__content>div {
+            background-color: white;
+        }
+
+        .feedback .card {
+            padding: 30px 35px;
+            gap: 20px;
+            border-radius: 5px;
+
+            @media screen and (max-width: 768px) {
+                gap: 10px;
+            }
+        }
+
+        .star__img {
+            width: 22px;
+            height: 22px;
+        }
+
+        .feedback .card__content {
+            color: #737373;
+            font-weight: 600;
+            line-height: 24px;
+        }
+
+        .content__reviewer {
+            display: grid;
+            grid-template-columns: 50px 95px;
+            grid-template-rows: 10px 10px;
+            grid-template-areas: 'image link'
+                'image profession';
+            column-gap: 15px;
+
+            @media screen and (max-width: 768px) {
+                grid-template-rows: 5px 5px;
+            }
+        }
+
+        .reviewer__img {
+            grid-area: image;
+            width: 50px;
+            height: 50px;
+        }
+
+        .reviewer__link {
+            grid-area: link;
+            color: var(--btn-primary-bg);
+            font-weight: 600;
+            line-height: 24px;
+        }
+
+        .reviewer__profession {
+            grid-area: profession;
+            font-weight: 600;
+            line-height: 24px;
+        }
+
+        .team {
+            background-color: var(--page-bg);
+        }
+
+        .team .container__heading {
+            margin: 80px 0;
+
+            @media screen and (max-width: 768px) {
+                margin-bottom: 45px;
+            }
+        }
+
+        .team .heading__section {
+            text-transform: initial;
+
+            @media screen and (max-width: 768px) {
+                max-width: 205px;
+            }
+        }
+
+        .team .paragraph__section {
+            @media screen and (max-width: 768px) {
+                max-width: 265px;
+            }
+        }
+
+        .team__content {
+            color: black;
+            display: grid;
+            gap: 30px;
+            grid-template-columns: 325px 325px 325px;
+            grid-template-rows: 370px;
+            margin-bottom: 80px;
+
+            @media screen and (max-width: 768px) {
+                grid-template-columns: 325px;
+                grid-template-rows: 370px 370px 370px;
+                margin-bottom: 95px;
+            }
+        }
+
+        .team__content>div {
+            background-color: white;
+        }
+
+        .team .card {
+            padding: 0;
+            gap: 0;
+        }
+
+        .card__photo {
+            width: 325px;
+            height: 230px;
+
+            @media screen and (max-width: 768px) {
+                width: 285px;
```

Comment:
avoid fixed width

Reply:
Is it possible to use "max-width" instead of "width" in this case?